wire-runtime-swift/src/main/swift/ProtoCodable/ProtoWriter.swift

```diff
@@ -59,10 +64,15 @@ public final class ProtoWriter {
 
     init(
         data: WriteBuffer = .init(),
-        outputFormatting: ProtoEncoder.OutputFormatting = []
+        outputFormatting: ProtoEncoder.OutputFormatting = [],
+        protoSyntax: ProtoSyntax = .proto2
```

Comment:
nit: can we name this `rootMessageSyntax` or `rootMessageProtoSyntax` so that it's clear this isn't the syntax used for all messages that are written out?

Reply:
I'll update to `rootMessageProtoSyntax`